cookbooks/aws-parallelcluster-slurm/recipes/update/clear_slurm_accounting.rb

```diff
@@ -23,3 +23,12 @@
   supports restart: false
   action %i(disable stop)
 end
+
+bash "Remove existing cluster name state file" do
```

Comment:
This logic is duplicated. What about reducing code duplication by defining this logic into a function and call that function?